light-client/ibc-client-starknet-types/src/encoding/impls/client_state.rs

```diff
@@ -21,22 +22,30 @@ delegate_components! {
                     symbol!("latest_height"),
                     EncodeLengthDelimitedProtoField<1, UseContext>,
                 >,
+                // EncodeField<
+                //     symbol!("chain_id"),
+                //     EncodeLengthDelimitedProtoField<2, UseContext>,
+                // >,
```

Comment:
The commented out part are done the same way as [`EncodeStarknetConsensusState`](https://github.com/informalsystems/ibc-starknet/blob/848eab5fa9d86caf6c1bf48b6ea983f72c90f52f/light-client/ibc-client-starknet-types/src/encoding/impls/consensus_state.rs) and [`EncodeStarknetHeader`](https://github.com/informalsystems/ibc-starknet/blob/848eab5fa9d86caf6c1bf48b6ea983f72c90f52f/light-client/ibc-client-starknet-types/src/encoding/impls/header.rs). Not sure why this doesn't work.